sklearn/datasets/lfw.py

```diff
@@ -146,7 +146,7 @@ def _load_imgs(file_paths, slice_, color, resize):
     if not color:
         faces = np.zeros((n_faces, h, w), dtype=np.float32)
     else:
-        faces = np.zeros((n_faces, h, w, 3), dtype=np.float32)
+        faces = np.zeros((n_faces, int(h), int(w), 3), dtype=np.float32)
```

Comment:
I guessed that the `h` and `w` in `not color` branch were getting converted to `int` in the upper `if resize is not None:` block and the ones in the color branch weren't. Don't really know if it is true, but it removed the warning :|

They can be float right now I guess, but will give an error in the future. Should I remove the ints in front of them?